tests/TestOne/PackageA/package.json

```diff
@@ -4,7 +4,11 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "PackageB": "../PackageB"
+    "PackageB": "../PackageB",
+    "buildtool": "../buildtool"
+  },
+  "pjc": {
+    "build": "$buildtool__root/bin/buildtool"
```

Comment:
We see `buildtool` as a binary package (it includes ready to run executable at package's root and don't require build).